docs/source/api/apollo-server.md

```diff
@@ -43,7 +43,7 @@ const server = new ApolloServer({
 <tr class="required">
 <td>
 
-#### `typeDefs`
+##### `typeDefs`
```

Comment:
These apiref symbols were missing a header level and consequently were huge and appearing in the TOC 🤷‍♂️

Reply:
I didn't like the huge but I kinda liked them in the TOC since they are actually a big percentage of what's in the file? I dunno

Reply:
Gotcha. We can explore potentially styling that header level to a reasonable size so that these symbols _do_ appear in the TOC as a fast-follow